code/modules/mob/mob.dm

```diff
@@ -215,7 +215,7 @@
 			msg = blind_message
 		else if(T != loc && T != src) //if src is inside something and not a turf.
 			msg = blind_message
-		else if(T.lighting_object && T.lighting_object.invisibility <= M.see_invisible && T.is_softly_lit() && !in_range(T,M)) //if it is too dark, unless we're right next to them.
+		else if(T.is_softly_lit() && !in_range(T,M)) //if it is too dark, unless we're right next to them.
```

Comment:
Was this like a ghost thing?

Reply:
not sure, but setting the lighting underlays to max invisibility doesnt seem to do anything so i dont think anything can safely care about it anymore (not that anything appears to go below lightings invisibility anyways).
should i be checking for the clients lighting plane alpha?

Reply:
not sure, but setting the lighting underlays to max invisibility doesnt seem to do anything so i dont think anything can safely care about it anymore (not that anything appears to go below lightings invisibility anyways).
should i be checking for the clients lighting plane alpha?

Reply:
yeah that seems sane, should do that

Reply:
done